WAL/WAL/Screen/History/View/HistoryTableViewCell.swift

```diff
@@ -228,26 +228,33 @@ class HistoryTableViewCell: UITableViewCell {
     }
     
     internal func setData(_ data: HistoryData) {
-        postId = data.postID
-        
-        isContentHidden = data.hidden ?? false
+        postId = data.reservationID
+        if data.showStatus == "OPEN" {
+            isContentHidden = false
+            coverView.isHidden = true
+        } else {
+            isContentHidden = true
+            coverView.isHidden = false
+        }
```

Comment:
```suggestion
extension UITableViewCell {
        enum ContentShowStatus: String {
                case open          = "OPEN"
                case closed       = "CLOSED"
        }
}

.....

        let showStatus = ContentShowStatus(rawValue: data.showStatus) ?? .open
        isContentHidden = showStatus == .open ? false : true
        coverView.isHidden = showStatus == .open ? true : false 
```

이런 식으로 enum으로 관리해서 로우한 값이 최대한 사용되지 않도록 수정할 수 있을 것 같고 삼항 연산자로 좀 더 짧은 구문으로 작성할 수 있을 것 같습니다!!